tests/VeresOne.spec.js

```diff
@@ -1,6 +1,5 @@
 const nock = require('nock');
 const chai = require('chai');
-chai.use(require('dirty-chai'));
```

Comment:
Ah wait, so this is crucial, let's not remove it. It turns out that because of Javascript and Mocha shenanigans, if you invoke various chai tests like `to.exist` without dirty chai, it fails to throw an error when it's actually wrong.